third_party/service-catalog/manifests/catalog/templates/etcd-pvc.yaml

```diff
@@ -0,0 +1,18 @@
+---
+# Source: catalog/templates/etcd-pvc.yaml
+
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: release-name-catalog-etcd
+  labels:
+    app: release-name-catalog-etcd
+    chart: "catalog-0.1.43"
+    release: "release-name"
```

Comment:
Why is this label required? (I'm realizing that these were probably generated by Tiller. Do we need Tiller to install service catalog?)

Reply:
The labels were generated using helm, based on the chart template provided by service catalog (https://github.com/google/kf/blob/master/third_party/service-catalog/charts/catalog/templates/etcd-pvc.yaml). We don't use Tiller to install service catalog, we just `kubectl apply` the yaml files directly.

If you're curious, this is how I generated the manifest yaml files: https://blog.giantswarm.io/what-you-yaml-is-what-you-get/